src/generic_core/uproxy_core.ts

```diff
@@ -100,6 +92,7 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
   }
 
   private pendingNetworks_ :{[name :string] :social.Network} = {};
+  private portControlSupport = false;
```

Comment:
I think this should be a tri-state (true, false, or pending), with corresponding UI.  (Also, private fields should have a trailing underscore).